functions.php

```diff
@@ -235,6 +235,44 @@ function custom_class( $classes, $field ) {
 	return $classes;
 }
 
+/**
+ *
+ * Update confirmation message for Gravity Forms.
+ */
+add_filter( 'gform_confirmation', 'custom_confirmation', 10, 3 );
+
+/**
+ *
+ * Return custom confirmation message for Gravity Forms.
+ *
+ * @param string $confirmation The default confirmation message.
+ * @param array  $form The form properties.
+ * @param array  $entry The form data.
+ *
+ * @return string The custom confirmation message.
+ */
+function custom_confirmation( $confirmation, $form, $entry ) {
```

Comment:
What do you think to change this function name to a more specific name? eg. `gf_custom_confirmation`

Reply:
I 100% agree with you! I used this because of [this function](https://github.com/greenpeace/planet4-master-theme/blob/master/functions.php#L220) that was also not very specific, but I've [updated both of them](https://github.com/greenpeace/planet4-master-theme/pull/1771/commits/0c659f1b5c375af6bd06a83e87d77484d71a6920) now 🙂